src/map.cpp

```diff
@@ -2229,20 +2229,15 @@ void MMVManip::initialEmerge(v3s16 blockpos_min, v3s16 blockpos_max,
 			continue;
 
 		bool block_data_inexistent = false;
-		try
 		{
 			TimeTaker timer2("emerge load", &emerge_load_time);
 
-			block = m_map->getBlockNoCreate(p);
-			if(block->isDummy())
+			block = m_map->getBlockNoCreateNoEx(p);
+			if (block == nullptr || block->isDummy())
```

Comment:
`!block` is faster to read and more common in our code base

Reply:
Good to know. There are more of them. Going to fix them all.